chain/client/src/test_utils.rs

```diff
@@ -1592,6 +1592,26 @@ impl TestEnv {
     pub fn get_runtime_config(&self, idx: usize, epoch_id: EpochId) -> RuntimeConfig {
         self.clients[idx].runtime_adapter.get_protocol_config(&epoch_id).unwrap().runtime_config
     }
+
+    /// Process a transaction, process chunks for one epoch length and return the execution outcome.
+    pub fn execute_tx(
+        &mut self,
```

Comment:
I'd maybe split this into a separate PR, there's a couple of questions here, like why we need to process epoch, or why do we pass in signer instead of account ID. Basically, if we want to bikeshed the good API here, let's do a separate PR. If we want to have something just for this PR, than this looks good, but let's not forget to revisit after merging -- suboptimal test helpers are a huge hazard.

Reply:
Yeah, let's refactor a bit in a separate PR. Also because this PR here depends on NEP decisions and I want a better test infrastructure asap, to also write test for other implementations.

Created #7217 with a first suggestion, please bikeshed on that PR :)